### ISSUE TITLE
Fix warning during training in dataset

### ISSUE THREAD

**Issue by lasse2507a** [P10-Natural-Language-Processing/Gut-Instincts#35]
UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  sample["weight"] = torch.tensor(sample["weight"], dtype=torch.float)
